libs/core/src/lib/table/directives/table-cell-checkbox.directive.ts

```diff
@@ -0,0 +1,10 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+
+@Directive({
+    selector: '[fdTableCellCheckbox], [fd-table-cell-checkbox]'
+})
+export class TableCellCheckboxDirective {
+    /** @hidden */
+    @HostBinding('class.fd-table__cell--checkbox')
+    fdTableCellClass: boolean = true;
+}
```

Comment:
I think it will be better to keep this as a property `@Input() checkbox: boolean = false` on `fd-table-cell`. So on examples it would look like that
`<th fd-table-cell [checkbox]="true">`